### ISSUE TITLE
Bug server-side data

### ISSUE THREAD

**Issue by dcorb** [jaydenseric/graphql-react#60]
Server-side query is not loading data for me.
I can't locate the bug, sorry for the bad description of the bub

To replicate, clone https://github.com/jaydenseric/graphql-react-examples
run `npm install graphql-react@latest`
and it will break
`cacheValue` will be always undefined.

**Comment by jaydenseric** [comment]
I don't think there is anything wrong with `graphql-react` regarding SSR, because I'm using the latest version to query the GitHub GraphQL API to get the release information for this route and it's working ok:

https://ruck.tech/releases

I hesitate to spend much effort debugging that upgrade because soon I will likely totally replace [Next.js](https://nextjs.org) and [Vercel](https://vercel.com) in the https://github.com/jaydenseric/graphql-react-examples project to use [Ruck](https://ruck.tech) and [Fly.io](https://fly.io) instead, which will be much cleaner.

**Comment by dcorb** [comment]
fair enough. Maybe others can chime in, if they encounter the same problem.
Good luck with Ruck! Very interesting project 👀
